semantic_conventions/trace/tls.yaml

```diff
@@ -0,0 +1,204 @@
+groups:
+  - id: tls
+    prefix: tls
+    type: span
```

Comment:
do we really need spans for TLS/SSL?
-  Could it be expressed with events or attributes on other spans?
- If span is necessary, what should be the kind and name of such span? 
- Do we need something that represents certificate validation result or we're going to use span status for it?

Reply:
> do we really need spans for TLS/SSL?

The attribute conventions created here are not suggesting any of those solutions. But based on @jsuereth's [comment](https://github.com/open-telemetry/semantic-conventions/pull/21#discussion_r1195427670) and your questions here this should be something added to the description of the section. To only have one discussion I will answer your questions above together with replying to @jsuereth's comment.